code/modules/surgery/bodyparts/head.dm

```diff
@@ -43,6 +43,8 @@
 	var/hair_alpha = 255
 	/// Is the hair currently hidden by something?
 	var/hair_hidden = FALSE
+	/// Lazy initialized hashset of all hair masks that should be applied
```

Comment:
hashset? you mean assoc list?

Reply:
An associated list in which all values are associated to "TRUE", in this case for fast insertion without creating duplicates.

Reply:
I'm complaining about the term because we barely use it but then again the only use in codebase is me from 4 years ago so whatever maybe it's fine.

this is a fine pattern but it isn't really something that should be optimized, especially because there's a chance this will be slower then |= due to the low expected size of this list. not worth profiling either way though just worth keeping in mind. this is very much a cold path.